reliability-engineering/terraform/modules/concourse-monitoring/grafana.tf

```diff
@@ -97,3 +97,29 @@ resource "aws_ecs_service" "concourse_grafana" {
     subnets         = var.private_subnet_ids
   }
 }
+
+provider "grafana" {
+  url  = "https://grafana.${local.monitoring_domain}"
+  auth = "owner:${random_string.concourse_grafana_admin_password.result}"
+}
+
+resource "grafana_data_source" "prom_data_source" {
```

Comment:
do these grafrana resources need some kind of "depends_on" to wait for the grafana to actually exist? .... maybe not much of an issue in reality since they _do_ exists right now

Reply:
I have a feeling we may already have a bootstrapping issue in this type of area with the concourse provider etc. But yeah maybe we should revisit these